generate.py

```diff
@@ -278,6 +284,14 @@ def check_positive(x):
     k_ = int(args.top_k) if args.top_k else None
     tempo_ = int(60 * 1e6 / int(args.tempo)) if args.tempo else 512820
 
+    if args.midi_input_file:
+        midi_parser_output = midi_parser(args.midi_input_file)
+        tempo_ = midi_parser_output[2]
+        midi_input_file = (midi_parser_output[1])[0:args.midi_file_tokens] if args.midi_file_tokens else midi_parser_output[1]
+          
+    else:
+      midi_input_file = ["<start>"]
+    
     music_transformer = load_model(args.path_to_model)
-    generate(model_=music_transformer, inp=["<start>"], save_path=args.save_path,
+    generate(model_=music_transformer, inp=midi_input_file, save_path=args.save_path,
```

Comment:
please change this block to reflect the suggested changes to the argument names from parser.add_argument above

Reply:
Will do

Reply:
Done